src/modules/renderMeals.js

```diff
@@ -0,0 +1,71 @@
+const { addLike, getAndUpdateLikes } = require('./addLike');
+const { getCounts } = require('./getCount');
+const { handleModalToggle } = require('./modal');
+const { fetchMeal } = require('./renderDetails');
+
+const home = document.querySelector('#home');
+const homeLink = document.querySelector('#homeLink');
+
+// involvement API
+/* eslint-disable operator-linebreak */
```

Comment:
- [ ] linters are here to help you keep a clean code base, disabling it does not help, instead try solving the error